integration/src/main/java/scripts/testscripts/CreateAzureIp.java

```diff
@@ -0,0 +1,29 @@
+package scripts.testscripts;
```

Comment:
I wrote a new 'unit' test that uses the real underlying services, and tests successful IP creation in azure. It is disabled by default because the credentials are not pushed, but it should be nice to be able to verify the code does indeed successfully create cloud resources as we iterate. 

Unfortunately, I did this at the level of a step/flight, and was unable to get it working through the `controlledResourceService` itself in tests due to underlying SAM complaining, at which point I punted. I left the second test there for reference with the above summarized. 

https://github.com/DataBiosphere/terra-workspace-manager/pull/457/files#diff-78d55c5069aedd76020b4b2a3638c747492d66415f806adff8a1e49aea9c3a6aR74